projects/1.20/assets/refurbished-furniture/refurbished_furniture/lang/zh_cn-base.json

```
@@ -0,0 +1,211 @@
+    "filterCategory.refurbished_furniture.food": "食物",
+    "filterCategory.refurbished_furniture.food.desc": "食物和烹饪原料",
+    "filterCategory.refurbished_furniture.items": "物品",
+    "filterCategory.refurbished_furniture.items.desc": "所有物品，锅铲、平底锅、吐司等等",
+    "death.attack.refurbished_furniture.ceiling_fan": "%1$s被吊扇切成了碎片",
+    "death.attack.refurbished_furniture.ceiling_fan.player": "%1$s被吊扇切成了碎片",
+    "block.refurbished_furniture.workbench": "工作台",
+    "block.refurbished_furniture.frying_pan": "平底锅",
+    "block.refurbished_furniture.post_box": "邮筒",
+    "block.refurbished_furniture.doorbell": "门铃",
+    "block.refurbished_furniture.recycle_bin": "回收桶",
+    "block.refurbished_furniture.plate": "盘子",
+    "block.refurbished_furniture.stone_stepping_stones": "石头踏脚石",
+    "block.refurbished_furniture.granite_stepping_stones": "花岗岩踏脚石",
+    "block.refurbished_furniture.diorite_stepping_stones": "闪长岩踏脚石",
+    "block.refurbished_furniture.andesite_stepping_stones": "安山岩踏脚石",
+    "block.refurbished_furniture.deepslate_stepping_stones": "深板岩踏脚石",
+    "block.refurbished_furniture.television": "电视机",
+    "block.refurbished_furniture.computer": "电脑",
+    "block.refurbished_furniture.door_mat": "门垫",
+    "block.refurbished_furniture.milk": "奶",
+    "item.refurbished_furniture.spatula": "锅铲",
+    "item.refurbished_furniture.knife": "刀",
+    "item.refurbished_furniture.package": "包裹",
+    "item.refurbished_furniture.wrench": "扳手",
+    "item.refurbished_furniture.television_remote": "电视遥控器",
+    "item.refurbished_furniture.bread_slice": "面包片",
+    "item.refurbished_furniture.toast": "吐司",
+    "item.refurbished_furniture.sweet_berry_jam": "甜浆果果酱",
+    "item.refurbished_furniture.sweet_berry_jam_toast": "甜浆果果酱吐司",
+    "item.refurbished_furniture.glow_berry_jam": "发光浆果酱",
+    "item.refurbished_furniture.glow_berry_jam_toast": "发光浆果酱吐司",
+    "item.refurbished_furniture.sea_salt": "海盐",
+    "item.refurbished_furniture.wheat_flour": "小麦粉",
+    "item.refurbished_furniture.dough": "面团",
+    "item.refurbished_furniture.cheese": "奶酪",
+    "item.refurbished_furniture.cheese_sandwich": "奶酪三明治",
+    "item.refurbished_furniture.cheese_toastie": "烤奶酪三明治",
+    "item.refurbished_furniture.raw_vegetable_pizza": "生蔬菜披萨",
+    "item.refurbished_furniture.cooked_vegetable_pizza": "熟蔬菜披萨",
+    "item.refurbished_furniture.vegetable_pizza_slice": "蔬菜披萨片",
+    "item.refurbished_furniture.raw_meatlovers_pizza": "生肉披萨",
+    "item.refurbished_furniture.cooked_meatlovers_pizza": "熟肉披萨",
+    "item.refurbished_furniture.meatlovers_pizza_slice": "肉披萨片",
+    "container.refurbished_furniture.workbench": "工作台",
+    "container.refurbished_furniture.drawer": "抽屉",
+    "container.refurbished_furniture.crate": "板条箱",
+    "container.refurbished_furniture.kitchen_drawer": "厨房抽屉",
+    "container.refurbished_furniture.cooler": "冷藏箱",
+    "container.refurbished_furniture.fridge": "冰箱",
+    "container.refurbished_furniture.freezer": "冰柜",

```

```suggestion
    "container.refurbished_furniture.freezer": "冷冻室",
```
建议改为“冷冻室”，实际表现是两格高的冰箱的下方那格，全文通查
冰柜一般指的是小卖部里那些用来装冰棍雪糕的冷藏柜子，和这个不太符合